src/logic/operate.js

```diff
@@ -0,0 +1,32 @@
+const Big = require('big.js');
+
+export function operate(numberOne, numberTwo, operation) {
+  let result = 0;
+  const x = new Big(numberOne);
+  const y = new Big(numberTwo);
+  switch (operation) {
+    case '+':
+      result = y.plus(x);
+      break;
+    case '-':
+      result = y.minus(x);
+      break;
+    case 'x':
+      result = y.times(x);
+      break;
+    case '÷':
+      result = y.div(x);
+      break;
+    case '%':
+      result = x.div(100);
+      break;
+    case '+/-':
+      result = x.times(-1);
+      break;
```

Comment:
[optional]
I don't think that it can generate the correct result for the following sequence.
```
10 + 5 +/- =
```
Should get 5 as answer, but your one will provide -5.